pkg/cmd/roachtest/spec/option.go

```diff
@@ -174,7 +174,7 @@ func (o terminateOnMigrationOption) apply(spec *ClusterSpec) {
 	spec.TerminateOnMigration = true
 }
 
-// TerminateOnMigration ensures VM is terminated in case GCE triggers a live migration.
+// TerminateOnMigration ensures VM is terminated in case GCE triggers a live upgrade.
 func TerminateOnMigration() Option {
```

Comment:
TerminateOnUpgrade?

Reply:
I think this is about VM migrations in GCP, reverted the change